src/components/web-components/rank-card/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { StyleSheet } from 'react-native';
+import { ITheme } from '../../../core/theme/itheme';
+
+export default (theme: ITheme) =>
+    StyleSheet.create({
+        container: {
+            backgroundColor: theme.colors.rankCard,
+            marginRight: '16.84%',
+            marginLeft: '16.84%',
+            marginTop: 37,
```

Comment:
It's a best practice not to use hardcoded values. 
We should build a function like `normalize` and then use `BASE_DIMENSION` based on that.

Now, on mobile we have the scale based on iPhone 11, but we should find a way to also normalize on web. We need to make a little research about best practices of normalize on web.

```
// based on iPhone 11's scale
const scale = SCREEN_WIDTH / 414;
```

Reply:
Also, as a rule of thumb in design, please build your items from top to to bottom, meaning that you should only use `marginBottom`. Only if necessary, add `marginTop`